django_guid/config.py

```diff
@@ -4,16 +4,16 @@
 
 class Settings(object):
     """
-    Settings for django_guid read from the Django settings in `settings.py`
+    Settings for django_guid read from the Django settings in `settings.py`.
+
     Inspired by django-auth-adfs from @jobec
     """
 
     def __init__(self):
-        self.GUID_HEADER_NAME = 'Correlation-ID'  # Required
+        self.GUID_HEADER_NAME = 'Correlation-ID'
         self.VALIDATE_GUID = True
 
         required_settings = [
-            'GUID_HEADER_NAME'
```

Comment:
Agreed. I was being a bit in between two thoughts, first I wanted it to be required, but then I added a default anyway. Thanks!